data/rulesets.js

```diff
@@ -717,16 +711,16 @@ exports.BattleFormats = {
 		},
 		onValidateTeam: function (team) {
 			let typeTable;
-			for (let i = 0; i < team.length; i++) {
-				let template = this.getTemplate(team[i].species);
-				if (!template.types) return [`Invalid pokemon ${team[i].name || team[i].species}`];
+			for (const [i, set] of team.entries()) {
```

Comment:
`entries()` seems unnecessary here.

Reply:
https://github.com/Zarel/Pokemon-Showdown/pull/4490/files#diff-791169e8e004f1c472a40eb12906ea15R717

I mean i'm fine with changing this to `team.indexOf(set)`, it's just that from previous PRs i had developed the assumption that `.entries()` was favorable